critiquebrainz/db/review.py

```diff
@@ -135,9 +135,11 @@ def get_by_id(review_id):
             "full_name": review["full_name"],
         }
         votes = db_revision.votes(review["last_revision"]["id"])
-        review["votes_positive_count"] = votes["positive"]
-        review["votes_negative_count"] = votes["negative"]
-        review["popularity"] = review["votes_positive_count"] - review["votes_negative_count"]
+        review["votes"] = {
+            "positive": votes["positive"],
+            "negative": votes["negative"],
+        }
+        review["popularity"] = review["votes"]["positive"] - review["votes"]["negative"]
```

Comment:
I closed #132 (see my comment there about why). This was done in a different context. I would agree to remove it here to avoid data duplication.